src/elements/ItemsScrollerContainerElement.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FunctionComponent } from 'react';
+
+const createScroller = ({ scrollerclassName, dataHorizontal, dataMousewheel, dataCenterfocus, id, className }: IProps) => ({
+    __html: `<div is="emby-scroller"
+    class="${scrollerclassName}"
+    ${dataHorizontal}
+    ${dataMousewheel}
+    ${dataCenterfocus}
+    >
+    <div
+        is="emby-itemscontainer"
+        ${id}
+        class="${className}"
+    >
+    </div>
+    </div>`
+});
+
+type IProps = {
```

Comment:
The same suggestions given in ItemsContainerElement can be applied here

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/2b40dbd50f39f56924b0d7ee6a3cab2dab38cbe4